apis/validator/sync_committee_subscriptions.yaml

```diff
@@ -0,0 +1,27 @@
+post:
+  operationId: "syncCommitteeSubscriptions"
```

Comment:
maybe better to be `prepareSyncCommitteeSubnet` matches better beacon committee subsction and generate dclient method would be more verbose  `client.prepareSyncCommitteeSubnet({params})`

Reply:
Updated but left as plural, given it's an array